sdk/python/tests/compiler/testdata/complex_coin.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import kfp.dsl as dsl
+
+
+class FlipCoinOp(dsl.ContainerOp):
```

Comment:
We should not directly use `ContainerOp` in new code

```python
@kfp.components.create_component_from_func
def flip_coin_op() -> str:
    import random
    return random.choice(["heads", "tails"])
```

Reply:
It's an example based on flip-coin. Should we change flip-coin too then?